cmd/listplans/list_plans_test.go

```diff
@@ -55,6 +55,64 @@ func (s *ListPlansCommandSuite) SetUpTest(c *gc.C) {
 	s.PatchValue(listplans.NewClient, listplans.APIClientFnc(s.mockAPI))
 }
 
+func (s *ListPlansCommandSuite) TestTabularOutput(c *gc.C) {
+	listPlans := &listplans.ListPlansCommand{
+		CharmResolver: &mockCharmResolver{
+			ResolvedURL: "series/some-charm-url",
+			Stub:        s.stub,
+		},
+	}
+	defer listPlans.Close()
+	ctx, err := cmdtesting.RunCommand(c, listPlans, "some-charm-url")
+	c.Assert(err, jc.ErrorIsNil)
+	c.Assert(cmdtesting.Stdout(ctx), gc.Equals,
+		`PLAN             	PRICE	DESCRIPTION                                       
+bob/test-plan-1  	     	Lorem ipsum dolor sit amet, consectetur adipiscing
```

Comment:
I think you'll see the text wrapping issue if you use a plan desc that has newline breaks in it (use yaml `|` instead if `>-`).